### DISCUSSION TITLE
Incorrect version set for resolvers in package.json

### DISCUSSION THREAD

**Discussion by felipeloge** [react-hook-form/react-hook-form#11474] [Q&A]
Hey,
I have a doubt about the version set in each package.json file for the resolvers (e.g. valibot).
I've installed the `@hookform/resolvers` package in my [NX](https://nx.dev/) project that uses module federation. However I'm getting the following error:
<img width="837" alt="Screenshot 2024-02-02 at 20 17 40" src="https://github.com/react-hook-form/react-hook-form/assets/5653024/b5d10915-9542-40fb-85cb-1ae5ea15c5f3">

So after taking a look in the packages/code, I can see the version is set as `1.0.0` in all packages (e.g. valibot):
https://github.com/react-hook-form/resolvers/blob/master/valibot/package.json#L4

The correct version should be the same from set in the root package.json, shouldn't be? (currently is 3.3.4).

**Comment by bigcakes** [comment]
Same issue here, this seems incorrect
